### PR TITLE
14 implement power spectrum normalization

### PR DESCRIPTION
**Pull request by DSilva27**

### CODE REVIEW

File: tests/test_power_spectrum_and_bfactor.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import torch
+from cryo_challenge.power_spectrum_utils import _centered_ifftn, compute_power_spectrum
+from cryo_challenge._preprocessing.bfactor_normalize import (
+    _compute_bfactor_scaling,
+    bfactor_normalize_volumes,
+)
+
+
+def test_compute_power_spectrum():
```
Comment (geoffwoollard) [inline tests/test_power_spectrum_and_bfactor.py:9]: I think this works because the FT of a gaussian is radially symmetric. And so a the power of a ray from the origin (`[: box_size // 2, 0, 0]`) is the same as the power of the 3D object.

I'd put that note in the docstring to explain the logic of this text

Comment (DSilva27) [reply tests/test_power_spectrum_and_bfactor.py:9]: This just gave me an idea for a new test. I will make the change and implement the new test.

File: tests/test_power_spectrum_and_bfactor.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    )
+
+    mean_squared_error = torch.mean((power_spectrum - power_spectrum_slice) ** 2)
+
+    assert mean_squared_error < 1e-3
+
+    return
+
+
+def test_bfactor_normalize_volumes():
```
Comment (geoffwoollard) [inline tests/test_power_spectrum_and_bfactor.py:39]: I'd put a docstring about how this test works. 

Does the volume need to be oscillatory, or was that just one example? Could it be random?

File: src/cryo_challenge/power_spectrum_utils.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import torch
+
+
+def _cart2sph(x, y, z):
```
Comment (geoffwoollard) [inline src/cryo_challenge/power_spectrum_utils.py:4]: I would write some docstring (perhaps for the whole file at the top) about the conventions assumed here (origin, etc)

**Review by geoffwoollard — CHANGES_REQUESTED** [review]
I suggested some docs

File: tests/test_power_spectrum_and_bfactor.py (hunk 2 above)
Comment (DSilva27) [reply tests/test_power_spectrum_and_bfactor.py:39]: A random volume would not be a good test because we would not know what to expect. It would be good to check that everything runs, but this test also takes care of that.

**Comment by DSilva27** [comment]
Finished doing the changes based on your comments.

**Comment by DSilva27** [comment]
Merging this PR as the requested changes were only doc-related changes.
